### ISSUE TITLE
Define doxygen group InitialTopographyModels

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1080]
#1063 adds a doxygen group `@ingroup InitialTopographyModels`. These groups need to be declared and documented in `doc/modules/groups.h`. This is still missing at this point.

@MFraters / @anne-glerum -- want to add this as a follow-up to #1063?

**Comment by MFraters** [comment]
Sure, will look into this

**Comment by MFraters** [comment]
I currently have as text:

```
/**
* @defgroup InitialTopographyModels Describing the properties of the initial topography of the domain
*
* A module for the definition of properties initial topography of the geometry.
* The goal if this module is to return the elelvation at a certain point on the
* surface and store and manange all information related to this porpose.
*/
```

@anne-glerum, what do you think?
I can also still add it to the pull request #1063, since it isn't merged yet.

**Comment by bangerth** [comment]
Leave it for a separate PR and let's get #1063 merged. The longer we wait, the more conflicts we usually accumulate and the longer it takes to chase the tester through it over and over again.

**Comment by bangerth** [comment]
Grammar: "A module for the definition of properties initial topography of the geometry."
manange -> manage

**Comment by gassmoeller** [comment]
Is this closed by #1083?

**Comment by MFraters** [comment]
Yes, I think so.
